stats/processor.go

```diff
@@ -0,0 +1,29 @@
+package stats
+
+import (
+	"bytes"
+	"github.com/melchiormoulin/graphite-writer-stats/output"
+	"go.uber.org/zap"
+	"strconv"
+)
+
+func process(logger *zap.Logger, metricPath string, componentsNb int) {
+	metric := GetMetric(logger, metricPath, componentsNb)
+	logger.Debug("metrics", zap.Any("metric", metric))
+	output.IncMetricPathCounter(metric.ExtractedMetric, metric.ApplicationName, string(metric.ApplicationType), strconv.FormatBool(metric.IsPrometheus))
+}
+
+func Process(logger *zap.Logger, dataPoint []byte, componentsNb int) bool {
+	metricPath, succeed := extractMetricPath(dataPoint)
+	if !succeed {
+		logger.Error("fail to convert to metricpath this datapoint", zap.ByteString("datapoint", dataPoint))
+		output.IncMetricPathErrorCounter(string(dataPoint))
+	}
+	process(logger, metricPath, componentsNb)
+	return succeed
+}
+
+func extractMetricPath(metric []byte) (string, bool) {
```

Comment:
this should check for valid metrics name and return an error to classify them in the error counters

OPT: Keep a track record of the last N "bad" metrics to help investigation to make them available, should we need them